src/data/home/Socials.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import {
+  FaInstagram,
+  FaFacebookSquare,
+  FaDiscord,
+  FaYoutube,
+  FaLink,
+} from "react-icons/fa";
+import { IoMail } from "react-icons/io5";
+
+export const SOCIALS = [
+  {
+    name: "Discord",
+    icon: <FaDiscord className="text-3xl isa-yellow-200" />,
```

Comment:
for the className, you can create a const javascript string variable called class containing the CSS. This is so we can adjust all styling to all classNames with a single edit, incase we need to make adjustments
```suggestion
    icon: <FaDiscord className={class} />,
```

Reply:
- icons are too small
- use yellow-100 instead